external/storm-kafka/README.md

```diff
@@ -242,6 +242,16 @@ use Kafka 0.8.1.1 built against Scala 2.10, you would use the following dependen
 
 Note that the ZooKeeper and log4j dependencies are excluded to prevent version conflicts with Storm's dependencies.
 
```

Comment:
Lets make a note that users have to explicitly state the dependency on kafka-core libs